SxGDriveTokenGenerator.py

```diff
@@ -3,25 +3,20 @@
 # Requirements:
 #   Python 3.7+
 #   Python Packages:
-#       oauth2client
+#       google-auth-httplib2 (not actually sure if needed or not)
+#       google-auth-oauthlib
 #       google-api-python-client
 
-import json
-import os
-import shutil
-import sys
+import json, os, shutil, sys, pickle

```

Comment:
Added `pickle` as it is used to store/retrieve the token received from InstalledAppFlow.